core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss

```diff
@@ -0,0 +1 @@
+//TODO: implement image styles
```

Comment:
Remove?

Reply:
will work on add this as part of a sub-taks.